src/main/resources/application.properties.j2

```diff
@@ -37,6 +37,8 @@ allow_graphql_mutation=false
 # redis.host =
 # redis.port =
 redis.ttl=-1
+redis.authentication.enabled=false
+redis.password=
```

Comment:
This should be commented out, with additional comments that this value will be read from environment variable **REDIS_PASSWORD**